src/components/menu/Menu.js

```diff
@@ -0,0 +1,25 @@
+import { headerLinks } from '../../constants/headerConst';
+import { Nav, NavLink, WorkButton, Wrapper } from './styledMenu';
+
+const Menu = (props) => {
+
+    return (
+        <Wrapper>
+            <Nav>
+                {headerLinks.map((link) => {
+                    return (
+                        <NavLink
```

Comment:
key prop is missing. I'd try the uniqueId library I installed so we don't have to use index